opensrp-goldsmith/src/main/java/org/smartregister/goldsmith/configuration/anc/AncMemberProfileRowDataProvider.java

```diff
@@ -49,8 +49,6 @@
 
     public void addMedicalHistoryData(List<ConfigurableMemberProfileRowData> rowDataList) {
         Date lastVisitDate = getLastVisitDate(commonPersonClient);
-        if (lastVisitDate == null)
```

Comment:
I think this might be null at some point?

Reply:
Yeah. Will add the check